Lib/asyncio/streams.py

```diff
@@ -978,26 +992,35 @@ def feed_data(self, data):
         #   messages :)
 
         # `offset` is the number of bytes from the beginning of the buffer
-        # where there is no occurrence of `separator`.
+        # where there is no occurrence of any separator.
```

Comment:
I'd leave the backticks here. Even though it's a list now, it still makes it clear that we're referring to the variable.

```suggestion
        # where there is no occurrence of any `separator`.
```